community.hachyderm.io/content/en/blog/posts/2024-06-09-Israel-Palestine-Conflict/index.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2024-06-09
+title: "The Israel-Palestine Conflict"
+linkTitle: "The Israel-Palestine Conflict"
+description: |
+  Blog Update and a Brief Discussion on the Israel-Palestine Conflict
+author: Hachyderm Moderation ([@Hachyderm](https://hachyderm.io/@Hachyderm))
+---
+
+Hey Hachyderm,
+
+Before we get started, we wanted to acknowledge that it's been a while since have used this forum to communicate. We have established a plan to cover a number of topics over the coming weeks
```

Comment:
Maybe something a *little* more informal, e.g.

> Before we get started, we wanted to acknowledge that it's been a while since we've written a blog post. Thank you for your patience! We've made a content plan and you should expect to see more activity through the blog starting this month covering a range of topics.